app/scripts/controllers/transactions/index.js

```diff
@@ -369,9 +369,11 @@ export default class TransactionController extends EventEmitter {
     txMeta.type = type;
 
     // ensure value
-    txMeta.txParams.value = txMeta.txParams.value
-      ? addHexPrefix(txMeta.txParams.value)
-      : '0x0';
+    if (txMeta.txParams.value && txMeta.txParams.value !== '0x') {
```

Comment:
I think we may want a more robust check here? You could still have a random string that could cause the same issue (I tried it with 
```"params": [
        {
            "data": "0x6abb9a460000000000000000000000000000000000000000000000000000000000000000",
            "value": **"JIEA"**,
            "gasPrice": "0x746a528800",
            "from": '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
            "to": "0xf7a2571ea4d6f478448551f7423085df61bd7cf4",
            "gas": "0xfb7e"
        }
    ],```
    and got the same error)
   
I'm assuming we want to just default to `0x0` for any invalid hex here? @danjm @darkwing @ryanml

Reply:
Good call @adonesky1 ; I think we can use `isHexString` here.

Reply:
there may be a better place to do this logic, namely in the normalizeAndValidateTxParams method and its constituents in app/scripts/controllers/transactions/lib/utils.js. This method is called anytime a transaction is added or updated in tx-state-manager.